### ISSUE TITLE
Error when changing to creative

### ISSUE THREAD

**Issue by Rocologo** [Gnat008/PerWorldInventory#218]
I get this error if I change my playermode from Survival to Creative. 

05.05 13:45:42 [Server] WARN [PerWorldInventory] [ECON] Unable to withdraw currency from bank of 'Rocologo': null

**Comment by angiechin** [comment]
I have the same error....  Running Spigot 1.11.2 and PWI latest build 1.10.1.  Our server have different worlds i.e normal Survival World, Plot World (Creative) and Skyblock World. We have enabled economy:true since we do not want player to have same money in Survival and Skyblock world.  Each time Player goes from Survival to Creative or to SkyBlock, the error msg show in console.

[08:47:20] [Server thread/WARN]: [PerWorldInventory] [ECON] Unable to withdraw currency from bank of 'Test': null
[08:47:20] [Server thread/INFO]: [Gmlist 1.5.5] Player Test is now in GameMode Survival.
[08:47:20] [Server thread/WARN]: [PerWorldInventory] [ECON] Unable to withdraw currency from bank of 'Test': null
[08:47:18] [Server thread/INFO]: Test issued server command: /home tiaga
[08:47:09] [Server thread/INFO]: [NoSpawnChunks] Unloaded 120 chunks. Took 182 ms.
[08:47:08] [Server thread/INFO]: [NoSpawnChunks] Unloading chunks in world MVSpawn...
[08:47:08] [Server thread/WARN]: [PerWorldInventory] [ECON] Unable to withdraw currency from bank of 'Test': null
[08:46:39] [Server thread/INFO]: [Gmlist 1.5.5] Player Test is now in GameMode Creative.
[08:46:37] [Server thread/WARN]: [PerWorldInventory] [ECON] Unable to withdraw currency from bank of 'Test': null
[08:46:32] [Server thread/INFO]: Test issued server command: /plot home

Also, occasionally Player's survival inventory and money is reset to 0 when they return to the Survival World from PlotWorld.....not sure if this has something to do with PWI and the error or something else.

My PWI config.yml and world.yml are enclosed.

[https://pastebin.com/V0uuH94q](url)
[https://pastebin.com/U4v7FPEn](url)

**Comment by Rocologo** [comment]
I agree except it my money disappears from my creative mode account. But only occasionally.
